asreview/webapp/src/Components/ExportDialog.js

```diff
@@ -86,7 +108,7 @@ const ExportDialog = (props) => {
         aria-labelledby="scroll-dialog-title"
         aria-describedby="scroll-dialog-description"
       >
-        <DialogTitle id="scroll-dialog-title">Export results {props.project_id}</DialogTitle>
+        <DialogTitle id="scroll-dialog-title">Export results/project {props.project_id}</DialogTitle>
```

Comment:
@MartijnUX We should think about the title of this screen (or maybe the dual functionality of this dialog at all.)